metrics/size.table/query.go

```diff
@@ -0,0 +1,19 @@
+package sizetable
+
+import (
+	"github.com/cashapp/blip"
+)
+
+func TableSizeQuery(set map[string]string, def blip.CollectorHelp) (string, error) {
+	query := `
+	SELECT table_schema AS db, table_name as tbl,
+		COALESCE(data_length + index_length, 0) AS tbl_size_bytes
+		FROM information_schema.TABLES`
+
+	if val := set[OPT_SCHEMA_FILTER]; val == "no" {
+		query = query + ` WHERE table_schema NOT IN ('performance_schema', 'information_schema', 'mysql', 'sys')`
+	}
+	query = query + `;`
```

Comment:
You don't need terminating `;` in code (unless you're doing multi-statements like `query := "foo; bar".  The `;` is an interactive shell thing, i.e. when humans input queries in the `mysql` shell.